optics-core/src/Optics/Iso.hs

```diff
@@ -74,14 +74,6 @@ module Optics.Iso
   , au
   , under
 
-  -- * Combinators
-  -- | The 'Optics.Re.re' combinator can be used to reverse an 'Iso':
-  --
-  -- @
-  -- 'Optics.Re.re' :: 'Iso' s t a b -> 'Iso' b a t s
-  -- @
-  , mapping
-
```

Comment:
We seem to have lost some documentation here. I'd suggest keeping the mention of `mapping` but in Haddock comments, like `re`.

Reply:
Fixed. I was confused, it looked like the above is "forgotten to update" documentation for `mapping`.
Resurrected it so *Combinators* section doesn't export anything, so it isnt' so confusing:

![Screenshot from 2020-02-10 14-46-20](https://user-images.githubusercontent.com/51087/74151128-21e60100-4c14-11ea-83d9-16a62f15f316.png)